internal/scaffold/helm/chart_test.go

```diff
@@ -100,12 +66,12 @@ func TestCreateChart(t *testing.T) {
 		},
 		{
 			name:          "repo without helm chart",
-			helmChartRepo: repoURL,
+			helmChartRepo: srv.URL(),
 			expectErr:     true,
 		},
 		{
 			name:             "non-existent version",
-			helmChart:        latest.repoAndName,
+			helmChart:        "test/" + chartName,
```

Comment:
If these are paths, they should be created using `filepath.Join`.

Reply:
These aren't paths. They're `<repo>/<chartName>` strings for remote paths. But there are some other paths that need `filepath.Join`